tools/rmats_both_bam.cwl

```diff
@@ -5,10 +5,13 @@ requirements:
   - class: ShellCommandRequirement
   - class: InlineJavascriptRequirement
   - class: DockerRequirement
-    dockerPull: 'xinglab/rmats:v4.1.2'
+    dockerPull: 'pgc-images.sbgenomics.com/danmiller/rmats-turbo:4.1.2'
```

Comment:
What was wrong with the original docker pull?

Reply:
ENTRYPOINT. ICA/cwltool does not respect ENTRYPOINT, CAVATICA does. I got rid of it because its confusing and the cwl authors say to avoid using dockers that have it.